packages/compiler/core/checker.ts

```diff
@@ -796,8 +792,16 @@ export function createChecker(program: Program): Checker {
    */
   function checkIntersectionExpression(node: IntersectionExpressionNode) {
     const optionTypes = node.options.map(getTypeForNode);
-
     const properties = new Map<string, ModelTypeProperty>();
+
+    const intersection: ModelType = createType({
+      kind: "Model",
+      node,
+      name: "",
+      properties: properties,
+      decorators: [], // could probably include both sets of decorators here...
```

Comment:
See this was already there but is there something we need to update here? or should this comment be removed

Reply:
I was just staring at this too. @bterlson thoughts?

Reply:
Sorry for crappy comments. I think this comment is very outdated and could be deleted. At some point before we decided & was sugar for ..., we might have copied decorators from the sources at this point. Comment can be removed.